src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-8150.mdx

```diff
@@ -19,25 +19,25 @@ security: []
 
 ## New features and improvements
 
-- Addition of AWS Lambda SDK instrumentation by @meiao in [1998](https://github.com/newrelic/newrelic-java-agent/pull/1998)
-- Reporting of Flyway migration events by @jtduffy in [2021](https://github.com/newrelic/newrelic-java-agent/pull/2021)
-- Add support for using an environment variable for config file location by @jtduffy in [2022](https://github.com/newrelic/newrelic-java-agent/pull/2022)
-- Support AWS Kinesis V1 and V2 SDKs by @obenkenobi in [2031](https://github.com/newrelic/newrelic-java-agent/pull/2031)
-- Addition of kafka-clients-node-metrics-3.7.0 Instrumentation module by @deleonenriqueta in [2039](https://github.com/newrelic/newrelic-java-agent/pull/2039)
-- Add instrumentation for glassfish-jul-extension logging library by @jasonjkeller in [2049](https://github.com/newrelic/newrelic-java-agent/pull/2049)
-- Java 23 support by @jasonjkeller in [2055](https://github.com/newrelic/newrelic-java-agent/pull/2055)
-- Support reporting of ECS Fargate Docker ids by @jtduffy in [2050](https://github.com/newrelic/newrelic-java-agent/pull/2050)
-- Actuator endpoint transaction naming for Spring Boot 3 by @jtduffy in [2077](https://github.com/newrelic/newrelic-java-agent/pull/2077)
+- Addition of AWS Lambda SDK instrumentation [1998](https://github.com/newrelic/newrelic-java-agent/pull/1998)
```

Comment:
Removing the contributor from the previous release notes.
We add the contributor on the release notes in GH, but on the docs website we don't.
This was an exception, so it is being corrected here.